src/objects.rs

```diff
@@ -565,7 +603,13 @@ unsafe impl OwnedStructureProperties for StructureTower {}
 // NOTE: keep impls for Structure* in sync with accessor methods in
 // src/objects/structure.rs
 
+unsafe impl HasStore for Ruin {
+    fn store_capacity(&self) -> u32 {
+        0 // no storeCapacity property
```

Comment:
That isn't true, unless you mean that we can't add to the storage?

Reply:
Yeah, that's right - they're like the structure version of tombstones, they may have resources inside them when they're created but you can only ever withdraw.